examples/vue-vanilla-ts-src/vite.config.ts

```diff
@@ -0,0 +1,16 @@
+import { join, resolve } from 'node:path'
+import viteFastifyPlugin from '@fastify/vite/plugin'
+import vuePlugin from '@vitejs/plugin-vue'
+
+export default {
+  root: join(import.meta.dirname, 'src', 'client'),
+  plugins: [
+    viteFastifyPlugin(), 
+    vuePlugin()
+  ],
+  build: {
+    // Forces Vite to use a top-level dist folder,
+    // outside the project root defined above
+    outDir: resolve('dist')
```

Comment:
Does calling `resolve('dist')` have the same result as `resolve(import.meta.dirname, 'dist')`? What happens if you `cd` into a subdirectory and then run `pnpm run build`?

Reply:
It should have the same result, yes. Without the fully qualified path, it considers `cwd`.